docs/content/developers/building-contributing.md

```diff
@@ -10,17 +10,41 @@ Alternately, you can see the latest build in action by visiting DDEV on [Gitpod]
 
 ## Testing a PR
 
-Each build of a PR has artifacts created in GitHub, so you can click the details of the [PR Build](https://github.com/drud/ddev/actions/workflows/pr-build.yml) test, choose the PR you want to work with, and download the artifacts you need.
+Each [PR build](https://github.com/drud/ddev/actions/workflows/pr-build.yml) creates GitHub artifacts you can use for testing, so you can download the one you need from the PR page, install it locally, and test using that build.
 
-After you download and unzip the appropriate binary, you can place it in your `$PATH`. The easiest way to do this if you’re using Homebrew is `brew unlink ddev` and then `unzip ddev.zip && chmod +x ddev && mv ddev /usr/local/bin/ddev`.
+Download and unzip the appropriate binary and place it in your `$PATH`.
 
-Verify the replacement worked via `ddev -v`. The output should be something like `ddev version v1.19.1-42-g5334d3c1`, instead of the regular `ddev version v1.19.1`.
+If you’re using Homebrew, start by unlinking your current binary:
 
-(On macOS these downloaded binaries are not signed, so you’ll want to `xattr -r -d com.apple.quarantine /path/to/ddev` in order to use them. The binaries on the master branch and the final binaries in any release *are* signed.)
+```
+brew unlink ddev
+```
+
+Next, unzip the binary you downloaded, make it executable, and move it to your bin folder:
+
+```
+unzip ddev.zip
+chmod +x ddev && mv ddev /usr/local/bin/ddev
```

Comment:
This may require sudo, `sudo mv ddev /usr/local/bin/ddev`

Reply:
Does it matter if I just did this and didn’t need `sudo`? M1, Ventura 13.0.1, `/opt/homebrew/bin/brew`.

Reply:
In the homebrew context on macOS, no sudo is needed, but if using /usr/local/bin on nearly anywhere else, it is.

Reply:
Updated!